src/Component/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { GoogleLogin } from 'react-google-login';
+import { useNavigate } from 'react-router-dom';
+// require('dotenv').config();
+
+const clientId = process.env.REACT_APP_CLIENT_KEY;
+
+function Login() {
+  const navigate = useNavigate();
+  const onSuccess = (res) => {
+    // console.log('Login Success:', res.profileObj);
+    navigate('/done');
```

Comment:
Will the main route have name as `/done`?

Reply:
NO i can give it like Dashboard. that is good?